Assets/Scripts/SpawnHandler/InputReader.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using UnityEngine;
+
+namespace SpawnHandler
```

Comment:
Move to another folder/namespace as InputReader is not only about spawning. Call folder like Inputs or smth like that